spotbugsTestCases/src/java/ghIssues/issue600/MyAutocloseable.java

```diff
@@ -0,0 +1,16 @@
+package ghIssues.issue600;
+
+public class MyAutocloseable implements AutoCloseable {
+    @Override
+    public void close() throws Exception {
+        System.out.println("close");
```

Comment:
Perhaps a logger instead of STDOUT?

Reply:
That class belongs to a testcase. Why use a logger? I prefer to keep them as basic as possible. I just didn't want to leave the method empty to avoid it being optimised away or something like that.

Reply:
Loggers are inherently more flexible and configurable than STDOUT. Consider a situation where you might want to capture test outputs and email them somewhere; it would be easier to configure that with a logger.

Also, as a result, tools like PMD will tend to flag usage of System.out.

Reply:
Sorry I should have been more precise.

This is one of the classes the unit tests are analysing. This code never actually runs. That's why it is in the spotbugsTestCases project. Have a look, there are a lot of System.out.printlns in that project.

While I am in general agree with you on using loggers in production and unit test, in this case I try to keep the code as minimal as possible and in order to achieve that, I avoid any dependencies. java.util.logging is not part of the Java base module, I don't want to include a module dependency for no good reason. I can't use java.lang.System.Logger since it doesn't exist in Java 8. I just need a dummy statement and System.out.println is easily understood by everyone.